render-wasm/src/main.rs

```diff
@@ -655,6 +655,26 @@ pub extern "C" fn set_modifiers() {
     });
 }
 
+#[no_mangle]
+pub extern "C" fn start_temp_objects() {
+    unsafe {
+        #[allow(static_mut_refs)]
+        let mut state = STATE.take().expect("Got an invalid state pointer");
+        state = Box::new(state.start_temp_objects());
+        STATE = Some(state);
```

Comment:
I would prefer if we mutated the current state instead of recreating the `Box`

Reply:
Again, you cannot do that because we need to pass the ownership.